src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  --header-color: #00acdc;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.header__container {
+  padding: 0 50px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: white;
+  flex-wrap: wrap;
+  gap: 60px;
+  max-width: 1200px;
+}
+
+.logo__img {
+  width: 100%;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px;
```

Comment:
The task specifies using `margin` instead of the `gap` property for flexbox indents. Please replace the `gap` property with appropriate `margin` settings.